code/modules/mod/mod_types.dm

```diff
@@ -196,6 +196,21 @@
 		/obj/item/mod/module/jetpack/advanced,
 	)
 
+/obj/item/mod/control/pre_equipped/praetorian
```

Comment:
Please move this to modular_skyrat/modules/blueshield/code

Reply:
Code made modular as requested